Documentation/guides/BuildProcess.md

```diff
@@ -1073,6 +1073,15 @@ server, the following MSBuild properties can be used:
     the value entered when `keytool` asks **Enter key password for
     $(AndroidSigningKeyAlias)**.
 
+    You can use the raw password here, however if you want to hide your password in logs
+    you can use a preview of env: or file: to point it to an Environment variable or 
```

Comment:
I think this is meant to say `prefix` rather than `preview`

Reply:
Fixed, thanks for the review :)